fails:IO.open propagates an exception raised by #close that is a StandardError
